modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -184,6 +284,12 @@ bool HttpSynchronization::trySyncFromUrl(std::string listUrl) {
             continue;
         }
 
+        //Check if the file exists in stored files in ossync, if so we ignore that download. 
+        if (std::find(_existingSyncedFiles.begin(),_existingSyncedFiles.end(), line) != _existingSyncedFiles.end()) {
```

Comment:
Very long line and in general we try not to do a lot of work inside the `if` statement, so:

```suggestion
        auto it = std::find(_existingSyncedFiles.begin() ,_existingSyncedFiles.end(), line);
        if (it != _existingSyncedFiles.end()) {
```